pennylane/measurements/classical_shadow.py

```diff
@@ -15,17 +15,18 @@
 """
 This module contains the qml.classical_shadow measurement.
 """
+import copy
 from collections.abc import Iterable
 
 import numpy as np
 
 import pennylane as qml
 from pennylane.wires import Wires
 
-from .measurements import MeasurementProcess, MeasurementShapeError, Shadow, ShadowExpval
+from .measurements import CustomMeasurement, MeasurementShapeError, Shadow, ShadowExpval
 
 
-def shadow_expval(H, k=1, seed_recipes=True):
+def shadow_expval(H, k=1, seed=None):
```

Comment:
Let's add this to the changelog since I would guess that it counts as a breaking change.

Reply:
Done!